src/tests/test_dte_data_models.py

```diff
@@ -909,15 +1008,21 @@ def test_init_fail_razon_ref_too_long(self) -> None:
                 ),
             )
         self.assertEqual(
-            assert_raises_cm.exception.errors(),
+            assert_raises_cm.exception.errors(
+                include_context=False,
+                include_input=False,
+                include_url=False,
+            ),
             [
                 {
                     'loc': ('razon_ref',),
-                    'msg': "('The maximum length allowed for `razon_ref` is 90', "
-                    "'Lorem ipsum dolor sit amet, consectetur adipiscing elit. "
-                    "Sed metus magna, ultricies sit amet dolor sed')",
+                    'msg': (
+                        "Value error, ('The maximum length allowed for `razon_ref` is 90', "
+                        "'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed metus "
+                        "magna, ultricies sit amet dolor sed')"
+                    ),
```

Comment:
Reformat to reduce diff size.

```suggestion
                    'msg': (
                        "Value error, "
                        "('The maximum length allowed for `razon_ref` is 90', "
                        "'Lorem ipsum dolor sit amet, consectetur adipiscing elit. "
                        "Sed metus magna, ultricies sit amet dolor sed')"
                    ),
```